components/generate-dataset.yaml

```diff
@@ -50,6 +50,7 @@ implementation:
 
       S3_DATADIR=s3://${s3_bucket}/${owner}/dataset/${project}/${experiment}/${dataset}/`date +%s`/
       aws s3 sync --no-progress datadir/ ${S3_DATADIR}
+      aws s3 cp --no-progress ${experiment}/manifest.tt ${S3_DATADIR}${experiment}/manifest.tt
```

Comment:
That's the job of the makefile right? Cause the manifest path will be different in different projects.